rdt/transformers/__init__.py

```diff
@@ -35,18 +46,48 @@ def _import_addons():
             for transformer in transformers:
                 module = transformer.rsplit('.', 1)[0]
                 if module not in sys.modules:
-                    importlib.import_module(module)
+                    module = importlib.import_module(module)
+                    for name, obj in inspect.getmembers(module):
+                        # Inject the addon transformer into this scope
+                        if inspect.isclass(obj) and issubclass(
+                                obj, BaseTransformer) and obj is not BaseTransformer:
+                            globals()[name] = obj
+                            __all__.append(name)
 
 
 _import_addons()
 
+
+def get_transformer_name(transformer):
+    """Return the fully qualified path of the transformer.
+
+    Args:
+        transformer:
+            A transformer class.
+
+    Raises:
+        ValueError:
+            Crashes when the transformer is not passed as a class.
+
+    Returns:
+        string:
+            The path of the transformer.
+    """
+    if inspect.isclass(transformer):
+        return transformer.__module__ + '.' + transformer.__name__
+
+    raise ValueError(f'The transformer {transformer} must be passed as a class.')
+
+
 TRANSFORMERS = {
-    transformer.__name__: transformer
+    get_transformer_name(transformer): transformer
     for transformer in BaseTransformer.get_subclasses()
 }
 
-globals().update(TRANSFORMERS)
-__all__.extend(TRANSFORMERS.keys())
+__all__.extend({
+    transformer.__name__
```

Comment:
why doesn't this use the `get_transformer_name` method? What happens if this list has multiple entries that are the same?